packages/headless/package.json

```diff
@@ -129,8 +129,10 @@
     "build:bundles": "node esbuild.mjs",
     "build:definitions": "tsc -p src/tsconfig.build.json -d --emitDeclarationOnly --declarationDir dist/definitions",
     "clean": "rimraf -rf dist/*",
-    "test": "vitest run",
-    "test:watch": "vitest",
+    "test": "vitest run --exclude \"src/integration-tests/**\"",
```

Comment:
The Headless integration tests are essentially used during development, while we don't have a UI to test against to ensure that the features we're implementing are working.

IMO they're rendered redundant by e2e tests, and could potentially be removed altogether.

For now, I suggest simply not running them in the CI, but still allowing them to be run locally, which is why I excluded them from the `test` (and for good measure `test:watch`) scripts, and added dedicated scripts for those tests.